STEP.md

```diff
@@ -0,0 +1,23 @@
+# Overview
```

Comment:
I am confused. We already have STEP.md on our branch... What's the baseline for this branch?

Reply:
I created a versioned branch `step-with-19.34.0`, which is just a copy of current `stripe:master`, and a baseline for this PR. Once I get ok on PR, I will merge into `step-with-19.34.0` and then push it to `getstep/stripe-java:step`. I feel more comfortable doing it this way.
I'm open to suggestions.